cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -541,21 +628,21 @@ insertConwayDelegCert ::
   SlotNo ->
   ConwayDelegCert StandardCrypto ->
   ExceptT SyncNodeError (ReaderT SqlBackend m) ()
-insertConwayDelegCert tracer cache network txId idx mRedeemerId epochNo slotNo dCert =
+insertConwayDelegCert _tracer cache network txId idx mRedeemerId epochNo slotNo dCert =
```

Comment:
maybe we can remove `_tracer` if it's not being used. 

totally not related to this PR and would probably require way too much work but it would be nice to have have an app wide monad transformer with all our heavily used configs eg: `Trace IO Text + SyncNodeParams` etc...